src/locales/en.json

```diff
@@ -373,7 +373,13 @@
     "dismiss_alert": "Dismiss alert",
     "shutter": "Shutter",
     "start_recording": "Start recording",
-    "finish_recording": "Finish recording"
+    "finish_recording": "Finish recording",
+    "document_types": "Documents you can use to verify your identity",
+    "selfie_video_actions": "Actions to record a selfie video",
+    "cross_device": {
+      "document_verification": "Steps required to continue document verification on your mobile",
```

Comment:
I think for document the string should be more generic. I would just put "Steps required to continue verification on your mobile" as it's more consistent with the title. Same for the the Spanish translation

Reply:
Yes, thinking about it I agree. Also cuts down on extra code needed just to display 2 mostly similar lines of text.